### PR TITLE
14672 Update approval type to be required field in restoration schema

### PR DESCRIPTION
**Pull request by argush3**
*Issue #:* /bcgov/entity#14672

*Description of changes:*

* Update `approvalType` in restoration schema a required field.
* Update tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

**Review by vysakh-menon-aot — APPROVED** [review]

File: src/registry_schemas/schemas/restoration.json [hunk 1]
```diff
@@ -49,6 +50,7 @@
                 "approvalType": {
                     "type": "string",
                     "title": "The Type of Approval, such as Approved by Court or Registrar.",
+                    "description": "In the case of a limited restoration extension or limited restoration to full filing, the approval type needs to the same value as the original limited restoration filing.",
```
Comment (severinbeauvais) [inline src/registry_schemas/schemas/restoration.json:53]: So the approval type is required for all 4 subtypes?

Comment (argush3) [reply src/registry_schemas/schemas/restoration.json:53]: yes

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by ketaki-deodhar — APPROVED** [review]
Looks good!

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**